ethexe/processor/src/handling/run.rs

```diff
@@ -16,178 +16,260 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{
-    host::{InstanceCreator, InstanceWrapper},
-    ProcessorConfig,
-};
-use core_processor::common::JournalNote;
-use ethexe_common::gear::Origin;
+//! # Chunked parallel program execution
+//!
+//! ## Overview
+//!
+//! This approach speeds up the processing of multiple programs in parallel.
```

Comment:
please be more precise with naming: e.g. what is chunk
and move short overview to the top

Reply:
Updated doc